infrastructure/src/main/java/org/corfudb/infrastructure/CorfuServer.java

```diff
@@ -142,6 +142,8 @@
                     + "              Layout endpoint to seed Management Server\n"
                     + " -n, --no-verify                                                          "
                     + "              Disable checksum computation and verification.\n"
+                    + " -N, --no-sync                                                          "
```

Comment:
nit: not sure what the conventions are but it might be a bit confusing as lower case and upper case 'n' have different implications.